core/arch/arm/plat-imx/imx_pl310.c

```diff
@@ -34,19 +39,59 @@ void arm_cl2_config(vaddr_t pl310_base)
 
 void arm_cl2_enable(vaddr_t pl310_base)
 {
-	uint32_t val;
+	uint32_t val __maybe_unused;
 
 	/* Enable PL310 ctrl -> only set lsb bit */
 	write32(1, pl310_base + PL310_CTRL);
 
+#ifndef CFG_PL310_SIP_PROTOCOL
 	/* if L2 FLZW enable, enable in L1 */
 	val = read32(pl310_base + PL310_AUX_CTRL);
 	if (val & 1)
 		write_actlr(read_actlr() | (1 << 3));
+#endif
 }
 
 vaddr_t pl310_base(void)
 {
 	return core_mmu_get_va(PL310_BASE, MEM_AREA_IO_SEC);
 }
 
+#ifdef CFG_PL310_SIP_PROTOCOL
+int pl310_enable(void)
+{
+	vaddr_t base = pl310_base();
+	arm_cl2_config(base);
```

Comment:
add empty line between last variable definitnio and first instruction.

Reply:
OK